helix-term/.gitignore

```diff
@@ -1 +1,2 @@
 /target
+test-explorer
```

Comment:
```suggestion
```

Reply:
This is necessary, because dummy folders are created under this directory when running `test_explorer`.

I will add comment to explain its purpose.